.classpath

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
don't commit IDE specific files to git please :) you can setup a global gitignore to avoid this for all your git repositories: 
http://egorsmirnov.me/2015/05/04/global-gitignore-file.html